src/test/run-tests.sh

```diff
@@ -2,7 +2,7 @@
 
 SCRIPTDIR="$(dirname ""$0"")"
 
-if [ ! -f "ssl/certificate.pem" ]; then
+if [ ! -f "$HOME/mozilla-iot/data/ssl/certificate.pem" ]; then
```

Comment:
We should probably have an environment variable like MOZIOT_HOME default to ${HOME}/mozilla-iot but allow it to be override. All of the bash scripts should probably use something like:
```
: ${MOZIOT_HOME:="${HOME}/mozilla-iot"}
```
or the slightly more verbose, but perhaps more easily understood
```
MOZIOT_HOME="${MOZIOT_HOME:=${HOME}/mozilla-iot}"
```
and then use ${MOZIOT_HOME} everywhere